node/blockAPI/metaBlock.go

```diff
@@ -65,6 +65,21 @@ func (mbp *metaAPIBlockProcessor) GetBlockByHash(hash []byte, withTxs bool) (*ap
 	return mbp.computeStatusAndPutInBlock(blockAPI, dataRetriever.MetaHdrNonceHashDataUnit)
 }
 
+// GetBlockByRound will return a shard APIBlock by round
```

Comment:
// GetBlockByRound will return a `meta` APIBlock by round

Reply:
Yes, fixed it here and for `GetBlockByHash` as well